platform/linux-generic/odp_ishm.c

```diff
@@ -125,6 +125,13 @@
  */
 #define ISHM_NB_FRAGMNTS (ISHM_MAX_NB_BLOCKS * 2 + 1)
 
+/*
+ * Memory reservations larger than ISHM_HUGE_PAGE_LIMIT (bytes) are allocated
+ * using huge pages (if available). Smaller reservations are done using normal
+ * pages to conserve memory.
+ */
+#define ISHM_HUGE_PAGE_LIMIT (64 * 1024)
```

Comment:
Might be worth moving this to the config file at some point? I can see this being a tunable parameter.